django/utils/feedgenerator.py

```diff
@@ -75,12 +123,24 @@ def __init__(
         feed_copyright=None,
         feed_guid=None,
         ttl=None,
+        stylesheets=None,
         **kwargs,
     ):
         def to_str(s):
             return str(s) if s is not None else s
 
+        def to_stylesheet(s):
+            return s if isinstance(s, Stylesheet) else Stylesheet(s)
+
         categories = categories and [str(c) for c in categories]
+
+        if stylesheets is not None:
+            if isinstance(stylesheets, (Stylesheet, str)):
+                raise TypeError(
+                    "stylesheets should be a list, not {stylesheets.__class__}"
```

Comment:
`f` is missing, so it's also untested
```suggestion
                   f"stylesheets should be a list, not {stylesheets.__class__}"
```

Reply:
That's what I get for testing the exception and not its message 🤦🏻 

Thanks!